README.md

```diff
@@ -124,6 +124,17 @@ For every test case, it is expected that
   - a testdata/golden/<test_name>/test.yaml exists to define
 template input params.`
 
+### For `abc templates describe`
```

Comment:
We should say what it does. Something like "The describe command downloads the template and prints out its description, and describes the inputs that it accepts."

It would also be nice to show a fully executed example, including the command output.

Reply:
Updated the description, also added example output.